webapp/channels/src/components/youtube_video/youtube_video.tsx

```diff
@@ -1,14 +1,17 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import cn from 'classnames';
```

Comment:
We usually import classnames as `classNames`. Can you change it just for coherence?